src/app/gallery/page.tsx

```diff
@@ -0,0 +1,99 @@
+'use client';
+
+import Masonry from '@/components/Masonry';
+
+const items = [
+  {
+    id: "1",
+    img: "https://picsum.photos/id/1015/600/900",
+    url: "https://example.com/one",
+    height: 400,
+  },
+  {
+    id: "2",
+    img: "https://picsum.photos/id/1011/600/750",
+    url: "https://example.com/two",
+    height: 250,
+  },
+  {
+    id: "3",
+    img: "https://picsum.photos/id/1020/600/800",
+    url: "https://example.com/three",
+    height: 600,
+  },
+  {
+    id: "4",
+    img: "https://picsum.photos/id/1020/600/800",
+    url: "https://example.com/three",
+    height: 200,
+  },
+   {
+    id: "5",
+    img: "https://picsum.photos/id/1015/600/900",
+    url: "https://example.com/three",
+    height: 400,
```

Comment:
can you get a few images from the current codefest site and use them here? instead of the placeholder images? @SadeeshaPerera